DSharpPlus.UnifiedCommands/DSharpPlus.UnifiedCommands.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <ProjectReference Include="..\DSharpPlus\DSharpPlus.csproj" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+  </ItemGroup>
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
This can be possible. But that would require to register each command by itself. The current client only allow adding a command by singles or overwriting by bulk.

Reply:
i assume you meant to reply to the comment above?

Reply:
Yes. Mb

Reply:
well, to answer: a config toggle for "overwrite commands?" should do. if it's disabled we just don't touch the commands